spec/classes/datadog_agent_spec.rb

```diff
@@ -1107,6 +1107,48 @@
               'content' => /^\ \ analyzed_spans:\n\ \ \ \ foo|bar: 0.5\n\ \ \ \ haz|qux: 0.1\n/,
               )}
             end
+
+            context 'with apm_enabled set to true and apm_automatic_scrubbing specified' do
+              let(:params) {{
+                :apm_enabled  => true,
+                :apm_automatic_scrubbing => {
```

Comment:
I would rename this to `apm_obfuscation` or `apm_trace_obfuscation`, since it's the name it takes in the config file.